### ISSUE TITLE
BUG: Static initialization blocks are removed when class organizer is run

### ISSUE THREAD

**Issue by rhnatiuk** [aljazsim/vs-code-typescript-class-organizer#51]
[Static initialization blocks](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Static_initialization_blocks) are simply disappearing (removed) when organizing a class. For example,

```typescript
class X {
	private static v: number;
	static {
		this.v = 42;
	}
}
```

will turn into

```typescript
class X {
	private static v: number;
}
```

after reorganization.

The correct behavior would be to move it, e.g., before the constructor.

**Comment by mann-david** [comment]
Are there any options to have tsco ignore the one file where I use a static block?  I'd hate to turn tsco completely off because of 1 file.

**Comment by aljazsim** [comment]
Hi David!

This has been fixed in version 1.0.27 I've just published. Give it a try!
